emanote/src/Emanote/Model/StaticFile.hs

```diff
@@ -37,4 +40,45 @@ staticFileSelfRefs =
     . R.unRoute
     . _staticFileRoute
 
+data StaticFileInfo where
+  StaticFileImage :: StaticFileInfo
+  StaticFileAudio :: StaticFileInfo
+  StaticFileVideo :: StaticFileInfo
+  StaticFilePDF   :: StaticFileInfo
+  StaticFileCode  :: Text -> StaticFileInfo
+  deriving stock (Eq, Show, Ord, Generic)
```

Comment:
Best to follow the naming convention of the project, so `StaticFileInfoImage` instead of `StaticFileImage`.